tests/test_on_data/test_gin_ecephys.py

```diff
@@ -214,7 +222,7 @@ class TestConverter(NWBConverter):
                     folder_path=str(DATA_PATH / "openephysbinary" / "v0.4.4.1_with_video_tracking"),
                     spikeextractors_backend=spikeextractors_backend,
                 ),
-                case_name=f"spikeextractors_backend={spikeextractors_backend}",
+                case_name=f"spikeextractors_backend_{spikeextractors_backend}",
```

Comment:
changed from `=` to `_` to allow to run tests interactively (having `=` in function name is not permitted)

Reply:
This works for me:
```
pytest tests/test_on_data/test_gin_ecephys.py::TestEcephysNwbConversions::test_recording_extractor_to_nwb_12_OpenEphys_spikeextractors_backend=True
```

Maybe you need some escape character on some terminals?

```
pytest tests/test_on_data/test_gin_ecephys.py::TestEcephysNwbConversions::test_recording_extractor_to_nwb_12_OpenEphys_spikeextractors_backend\=True
```

Reply:
yes, but I wanted to be able to instantiate and call the test class in interactive mode, and the `=` makes it impossible

Reply:
Can you show me a screenshot o the error that you are geeting (also to see what is your workflow for testing-debugging)?

Reply:
@alejoe91 To clarify, mostly out of curiosity, what is the command you used that caused this `=` vs `_` error?